diff -urN detect-old/src/SuperProbe/AsmMacros.h detect/src/SuperProbe/AsmMacros.h
--- detect-old/src/SuperProbe/AsmMacros.h	Tue Mar 27 08:15:39 2001
+++ detect/src/SuperProbe/AsmMacros.h	Tue Mar 27 08:17:38 2001
@@ -60,13 +60,17 @@
 /* $XFree86: xc/programs/Xserver/hw/xfree86/SuperProbe/AsmMacros.h,v 3.10.2.1 1999/06/17 16:23:57 hohndel Exp $ */
 
 #if defined(__GNUC__)
-#if defined(linux) && (defined(__alpha__) || defined(__powerpc__))
+#if defined(linux) && (defined(__alpha__) || defined(__powerpc__) || defined(__ia64__))
+#ifdef __ia64__
+#include <sys/io.h>
+#else /* !__ia64__ */
 #define inb _inb
 #define inw _inw
 #define inl _inl
 #define outb(p,v) _outb((v),(p))
 #define outw(p,v) _outw((v),(p))
 #define outl(p,v) _outl((v),(p))
+#endif /* !__ia64__ */
 #else
 #if defined(__sparc__)
 #ifndef ASI_PL
